packages/shared-ux/chrome/navigation/src/ui/components/panel/label_badge.tsx

```diff
@@ -25,14 +25,13 @@ export const LabelBadge = ({
   className?: string;
 }) => {
   const { euiTheme } = useEuiTheme();
-
   return (
     <EuiBetaBadge
       label={text}
       size="s"
       css={css`
         margin-left: ${euiTheme.size.s};
-        color: ${euiTheme.colors.text};
+        color: ${euiTheme.colors.textParagraph};
```

Comment:
I'm not sure we are actually using this component anywhere currently, but it could be somewhere in the navigation panel for solutions:

https://github.com/user-attachments/assets/cfe59210-5811-4aec-86ea-cd29a616e7ee